src/components/comments/CommentsComponent.jsx

```diff
@@ -76,12 +86,20 @@ const ViewComments = ({ comments, slug, user }) => (
                     <p className="card-text">{reply.body}</p>
                   </div>
                   <hr />
-                  {user === reply.author.username ? (
-                    <span>
-                      <Button className="card-link btn-comment" data-toggle="collapse" data-target={`#editReply-${reply.id}`}><i className="fas fa-edit" /></Button>
-                      <DeleteComment slug={slug} id={reply.id} />
-                    </span>
-                  ): null }
+                  <Row>
+                    <Col xs="auto" className="mt-2">
+                      <LikeComment comment={reply} slug={slug} /> 
+                    </Col>
+                    <Col xs="auto">
+                      {user === reply.author.username ? (
+                        <span>
+                          <Button className="card-link btn-comment" data-toggle="collapse" data-target={`#editReply-${reply.id}`}><i className="fas fa-edit" /></Button>
```

Comment:
Hello @IssaIan ,
You could consider putting the Button properties in new lines once they exceed 3 as agreed earlier.

Reply:
Done